dev-docs/source/GettingStartedWithPyCharm.rst

```diff
@@ -180,11 +181,32 @@ The ``pydevd`` package does not have to be installed on Python 2. As of 12/11/20
 Setting up PyCharm on Linux
 ###########################
 
-On Linux the instructions are identical to Windows except that :
+1. Use the native python interpreter (``/usr/bin/python2.7/python.exe``) rather than from ``<Mantid Source Directory>/external/src/ThirdParty/lib/python2.7/python.exe``
```

Comment:
`/usr/bin/python2.7/python.exe` is likely wrong. Can this be changed to simple `python2.7` to pick up the correct binary from `PATH`?

Reply:
👍 It is wrong, it should be `/usr/bin/python2.7` and it can't be changed to juts `python2.7` as PyCharm shows the full path.